ansible/run.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: local
+  connection: local
+  vars_files:
+      - runtime_vars.yml
+  roles:
+      - facts
+      - nodejs
```

Comment:
It would be great if the deployed app gets tested here or by the ``preferences-server`` role. Please refer to the example in the ``ansible-nodejs`` role's [README.md](https://github.com/idi-ops/ansible-nodejs/blob/master/README.md):

```
nodejs_app_host_address: 127.0.0.1
nodejs_app_tcp_port: 8081
nodejs_app_test_http_endpoint: /preferences/carla
nodejs_app_test_http_status_code: 200
nodejs_app_test_string: registry.gpii.net
```